micrometer-tracing-tests/micrometer-tracing-integration-test/src/main/java/io/micrometer/tracing/test/SampleTestRunner.java

```diff
@@ -356,6 +352,7 @@ void run(SampleRunnerConfig sampleRunnerConfig, ObservationRegistry observationR
                     MeterRegistry meterRegistry, SampleTestRunner sampleTestRunner) {
                 checkTracingSetupAssumptions(WAVEFRONT_OTEL, sampleTestRunner.getTracingSetup());
                 checkWavefrontAssumptions(sampleRunnerConfig);
+                assert sampleRunnerConfig.wavefrontToken != null;
```

Comment:
The `checkWavefrontAssumptions` method called immediately before this checks wavefrontToken is not null, but NullAway doesn't take that into account.